test/test_scanner.py

```diff
@@ -635,6 +647,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
         list(
             itertools.chain(
                 [
+                    (
+                        "https://koji.fedoraproject.org/koji/buildinfo?buildID=1444639",
```

Comment:
This is the wrong url.  `https://kojipkgs.fedoraproject.org//packages/perl/5.30.1/450.fc32/x86_64/` to go with `perl-5.30.1-450.fc32.x86_64.rpm`  (You can figure it out by going to the build link above, right clicking on the download link you want, and then splitting that url into two pieces, one for the filename and one for the rest)